factories/exporters.go

```diff
@@ -54,7 +53,6 @@ var defaultExporters = []component.ExporterFactory{
 	awscloudwatchlogsexporter.NewFactory(),
 	awsemfexporter.NewFactory(),
 	awskinesisexporter.NewFactory(),
-	awsprometheusremotewriteexporter.NewFactory(),
```

Comment:
Did we mean to remove this?

Reply:
Yeah, as mentioned in the PR description, it was previously deprecated, and has now been deleted from contrib in v0.58.0.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/13173 for context

Reply:
Ok I thought I still saw it in the documentation so that's why I asked.

Reply:
Added a comment to just point out the line in the exporter readme.